README.md

```diff
@@ -71,6 +73,73 @@ dataset
 }
 ```
 
+
+## Diagnose your data generated
+
+```python
+from fiddlecube import FiddleCube
+fc = FiddleCube(api_key="<api-key>")
+result = fc.diagnose([
+        {
+            "query": "What is the capital of France?",
+            "answer": "Paris",
+            "prompt": "You are an expert at answering hard questions.",
+            "context": [
+                "Paris is the capital of France."
+            ]
+        }
+    ]
+)
+result
+```
+
+```json
```

Comment:
IMO we can remove the output examples now that we have quite a bit of content.
Later, this can be a part of the docs hosted in fiddlecube.ai/docs with better content formatting.

Reply:
Sure makes sense